2025/Nizzi/Istruzioni.txt

```diff
@@ -0,0 +1,54 @@
+Scopo
```

Comment:
Puoi scrivere questo file in formato .md cosi' che viene visualizzato in modo piu' moderno?